edisyn/synth/novationastation/UIBuilder.java

```diff
@@ -107,94 +115,119 @@ private JComponent createGlobal(Color color) {
         return globalCategory;
     }
 
-    private JComponent createGeneral(Color color) {
+    private JComponent createGeneral(Color color)
+    {
         Category categoryGeneral = new Category(synth, "General", color);
 
         HBox hbox = new HBox();
         VBox vbox = new VBox();
-        vbox.add(createChooser("polyphony mode", POLYPHONY_MODE));
-        vbox.add(createChooser("keysync phase", KEY_SYNC_PHASE));
+        vbox.add(createChooser("Polyphony Mode", POLYPHONY_MODE));
+        var keySync = createChooser("Keysync Phase", KEY_SYNC_PHASE);
+        // set ranges for metric values
+        keySync.setMin(Boundaries.KEYSYNC_PHASE.getMin() + 1); // skipping first, being a non metric value ("Free running")
```

Comment:
hmm, actually doubting if this is the proper way to define the first value as a non-metric... Is it, @SeanLuke ?

Reply:
[This](https://github.com/eclab/edisyn/pull/84/commits/1d3174b7119e1bfbac338b100e2c3e73ff211a5d) is how it should like, I guess (including some method renaming though...)

Reply:
Every parameter in Edisyn's model has an absolute minimum and maximum, and it also has an optional metric minimum and metric maximum.  The metric min and max define the range of actual metric, versus categorical values.   In short Min <= metricMin <= metricMax <= max.    For example, if you have the values -1=OMNI, 0=OFF, 1..16=channels, and I dunno, 17=MPE, then you'd set the min = -1, max = 17, metricMin = 1 and metricMax = 16.   The purpose of this is to help mutators (like the hill-climber) do a better job.   

When you make a LabelledDial with a parameter, it sets the metric min and max to the same values as the min and max.  When you make a chooser or a checkbox, it just sets the min and max.    This usually suffices.  But if you create a chooser that has metric values in it, you might want to call getModel().setMetricMinMax("key", min, max).  Also if you make a LabelledDial with some non-metric values, you might want to call the same; or remove the metric values entirely with removeMetricMinMax(...)

Reply:
ack, thx for the hint, I updated code to (skip the non-metric) with "getModel().setMetricMinMax"